common.gradle

```diff
@@ -136,13 +132,11 @@ if(rootProject.useLocalDependencies) {
 
         if (rootProject.hasProperty("only_daydream")) {
             compile "org.gearvrf:backend_daydream:$gearvrfVersion"
```

Comment:
For the case where we use maven you could make the daydream backend's dependencies transitive:
```
            compile("org.gearvrf:backend_daydream:$gearvrfVersion") {
                transitive = true
            }
```
Then remove `` compile 'com.google.vr:sdk-base:1.30.0'``. This way at least one of the branches will no longer need updating.